src/Workspaces/VisualBasic/Portable/Extensions/SyntaxTreeExtensions.vb

```diff
@@ -29,22 +29,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
         End Function
 
         Private Function FindTriviaToLeft(nodeOrToken As SyntaxNodeOrToken, position As Integer) As SyntaxTrivia
+recurse:
             For Each child In nodeOrToken.ChildNodesAndTokens().Reverse()
-                If child.FullSpan.Start < position AndAlso position <= child.FullSpan.End Then
+                If (child.FullSpan.Start < position) AndAlso (position <= child.FullSpan.End) Then
                     If child.IsNode Then
-                        Return FindTriviaToLeft(child, position)
+                        nodeOrToken = child
+                        GoTo recurse
                     Else
                         Dim triviaList = child.GetLeadingTrivia().Concat(child.GetTrailingTrivia()).Reverse()
```

Comment:
@mattwar Just check we're on the same lines.
 Two separate `ForEach`, first `GetTrailingTrivia.Reverse` second `GetLeadingTrivia.Reverse`